src/js/angular/ttyg/templates/chat-item-detail.html

```diff
@@ -24,6 +24,14 @@
                     </button>
                     <copy-to-clipboard tooltip-text="ttyg.chat_panel.btn.copy_answer.tooltip"
                                        text-to-copy="{{answer.message}}"></copy-to-clipboard>
+                    <div ng-if="answer.tokenUsageInfo"
+                         class="btn btn-link btn-sm token-usage-info-btn"
```

Comment:
A tag which is not a button styled with button css classes. I know that you use it to just attach the popover handler to it, but still. Better use button and prevent its default click behavior,